tests/test_shapes.py

```diff
@@ -0,0 +1,75 @@
+import sys
+from pdb import set_trace as bp
+import rdflib
+import brickschema
+from bricksrc.namespaces import A, OWL, RDFS, SKOS, BRICK, SH, BSH, bind_prefixes
+import pyshacl
+
+schema_g = brickschema.Graph().parse('shacl/BrickShape.ttl', format='turtle')
+#ont_g = brickschema.Graph().parse('Brick.ttl', format='turtle')
+bind_prefixes(schema_g)
+
+prefixes = """
+@prefix brick: <https://brickschema.org/schema/Brick#> .
+@prefix : <http://example.com#> .
+"""
+
+base_data = prefixes + """
+:equip a brick:Equipment.
+:point a brick:Point.
+:loc a brick:Location.
+"""
+
+def test_no_relations():
+    data = base_data
+    data_g = rdflib.Graph().parse(data=data, format='turtle')
+    conforms, results_graph, results_text = pyshacl.validate(
+        data_g, shacl_graph=schema_g,
+    )
```

Comment:
If you use `brickschema.Graph` then the `g.validate()` method maps to the pyshacl function with the same arguments